src/pages/updates/whats-new/index.mdx

```diff
@@ -198,418 +198,4 @@ description: This changelog reflects the most recent changes to the design syste
-
-![Sketch Palettes plugin](images/sketch-palettes.png)
-
-<Caption>Sketch palettes plugin.</Caption>
-
-</Column>
-</Row>
-
-### Carbon Sessions
-
-Interested in getting some 1:1 time with the Carbon team? Carbon Sessions is a chance for anyone using the Carbon Design System to ask questions and get feedback. Sessions happen very Wednesday morning (Central Standard Time). Time slots are around 10 minutes, but feel free to book more than one if you need additional time.
-
-Sign up for a [Carbon Sessions time slot](https://github.com/carbon-design-system/design-system-website/wiki/Carbon-Sessions) if you're interested in chatting further!
-
-## January 2018
-
-### Components
-
-#### Dropdown
-
-We're excited to bring some added functionality that enhances our [Dropdown](/components/dropdown/code) component. Filtering, Multi-select, and Inline Dropdown are new additions to our basic Dropdown component.
-
-_Note: This component is currently only available in [React](http://react.carbondesignsystem.com/?selectedKind=ComboBox&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)._
-
-<Row>
-<Column colLg={8}>
-
-![example of Dropdown with Multi-Select](images/dropdown-new.gif)
-
-<Caption>Example of a dropdown with multi-select.</Caption>
-
-</Column>
-</Row>
-
-#### Small Toggle
-
-Small Toggle is a style variation of our traditional [Toggle](/components/toggle/code) component. It's more compact in size, which allows them to be used in Data Tables or inline with Form elements.
-
-<Row>
-<Column colLg={8}>
-
-![example of Small toggle](images/small-toggle.gif)
-
-<Caption>Example of small toggle.</Caption>
-
-</Column>
-</Row>
-
-#### Data Vis: Bar Graph
-
-Bar Graph is the newest addition to our Data Visualization collection. It comes in two variations: Bar Graph and Grouped Bar Graph. You can use this component to represent quantitative information, such as comparing the taco sales between Austin taco shops in the month of June.
-
-Read more about the usage of [Bar Graph](/data-visualization/bar-graph/usage), or check out our [CodePen](https://codepen.io/team/carbon/pen/OzBKKN) to view the live demo.
-
-<Row>
-<Column colLg={8}>
-
-![Bar Graph example](images/usage-bar-graph.png)
-
-</Column>
-</Row>
-
-#### Data Vis: Tooltip
-
-We have created several Data Visualization Tooltip variations to show the value for a specific data point or increment range. Tooltips appear on hover above or below a data representation, depending on the use case.
-
-Read more about the usage of [Data Vis Tooltips](/data-visualization/tooltip/usage), or check out our [CodePen](https://codepen.io/team/carbon/pen/rJWymp) to view the live demo.
-
-<Row>
-<Column colLg={8}>
-
-![small tooltip in a bar graph](images/usage-small-tooltip.png)
-
-<Caption>Small tooltip used within a bar graph.</Caption>
-
-</Column>
-</Row>
+See Carbon's GitHub release log for the full archive of version releases and past changelogs.
```

Comment:
```suggestion
See Carbon's GitHub [release log](https://github.com/carbon-design-system/carbon/releases) for the full archive of version releases and past changelogs.
```

Reply:
Lol I just added the link before you commented